config/locales/en.yml

```diff
@@ -672,8 +672,8 @@ en:
 
       rules_not_met:
         caption:
-          importing: Importing commodity %{commodity_code} from %{trade_country_name}
-          exporting: Exporting commodity %{commodity_code} from %{trade_country_name}
+          importing: Importing commodity %{commodity_code} from %{service_country_name} to %{trade_country_name}
+          exporting: Exporting commodity %{commodity_code} from %{service_country_name} to %{trade_country_name}
```

Comment:
Is this phrasing correct, eg

"Importing commodity 12345 from the UK to Italy"

and

"Export commodity 1234 from the UK to Italy"

That first translation looks like it should read "from Italy to the UK" ? Or "to the UK from Italy"